pyclient/tests/test_query.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+from tests.testbase import BaseTestCase
+
+
+class QueryTestCase(BaseTestCase):
+    def test_tail_update_static(self):
+        tbl = self.session.empty_table(10)
+        query = self.session.query(tbl).update(column_specs=["Col1=i + 1"]) \
+            .tail(5).update(column_specs=["Col2=i*i"])
+        rlt = query.exec()
+        self.assertEqual(10, rlt.size)
+        time.sleep(1)
+        rlt2 = query.exec()
+        self.assertEqual(rlt.size, rlt2.size)
+
+    def test_tail_update_ticking(self):
+        tbl = self.session.time_table(period=100000)
+        query = self.session.query(tbl).update(column_specs=["Col1=i + 1"]) \
+            .tail(5).update(column_specs=["Col2=i*i"])
+        rlt = query.exec()
```

Comment:
Looking at this example, I have two questions:
1) How do you handle multiple inputs to the query
2) How do you handle multiple outputs to the query
In both cases, it seems like what is here may have problems.